elyra/pipeline/processor.py

```diff
@@ -88,8 +89,14 @@ async def export(self, pipeline, pipeline_export_format, pipeline_export_path, o
 
 
 class PipelineProcessorResponse(object):
-    def __init__(self, run_url='', object_storage_url='', object_storage_path=''):
+    def __init__(self,
```

Comment:
Along the lines of my previous comment - in which I asked whether the platform indicator is present in the response - I think it would make sense to make this an abstract base class and let each platform processor define a concrete subclass.

In that case, `AirflowProcessorResponse` would _add_ a _required_ `git_url` to the constructor, whereas `KfpProcessorResponse` would simply subclass the ABC `PipelineProcessorReponse`.  The ABC would then define an "abstract attribute" named, say, `platform`, and each of the concrete classes would set `platform` accordingly.

Reply:
created new base class with suggested parameters and respective subclasses in each runtime processor